noir/noir-repo/docs/docs/how_to/how-to-oracles.md

```diff
@@ -20,6 +20,10 @@ This guide shows you how to use oracles in your Noir program. For the sake of cl
 
 For reference, you can find the snippets used in this tutorial on the [Aztec DevRel Repository](https://github.com/AztecProtocol/dev-rel/tree/main/code-snippets/how-to-oracles).
```

Comment:
Let's keep this PR focused on the Aztec docs and we can check with @signorecello and open another one for the noir docs.

Reply:
Done